src/summary/composer.rb

```diff
@@ -115,6 +119,21 @@ def get_resource_services(installed_services, resource_id)
       return collection
     end
 
+    def get_resource_deployment_name(provisioned_resource)
+      output = ""
+      if provisioned_resource.get_resource().respond_to?(:get_tags)
+        deployment_name = provisioned_resource.get_resource().get_tags()["dxDeploymentName"].to_s
+        output += "deployment name: #{deployment_name}\n" unless deployment_name.empty?
```

Comment:
@noahmmcgivern I didn't see a unit test to validate this addition. Similar feedback for the deploy config url.